projects/uitdatabank/docs/search-api/advanced-queries.md

```diff
@@ -0,0 +1,1020 @@
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers` `/organizers`
+
+**Possible values**
+
+Allowed values for both are any of the two-letter language ISO 639-1 codes, but the most used languages are: `nl` `fr` `de` `en`
+
+**Example**
+
+Retrieve all organizers that are fully translated in English:
+```
+GET /oragnizers/?q=completedLanguages:en
+```
+
+### created
+
+UiTdatabank keeps track of the creation dates of each document. This metadata for the creation date can be queried with the `created` parameter.
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers` `/organizers`
+
+**Possible values**
+
+Any ISO-8601 datetime, e.g. `2023-04-01T12:08:01+01:00`
+
+<!-- theme: info -->
+> Note that the `+` sign should be encoded for URLs (as `%2B`). Otherwise it will be interpreted as whitespace and the given date time will be considered invalid.
+
+**Example**
+
+Get all events and places that are created since the start of 2022:
+```
+GET /offers/?q=created:[2022-01-01T00\:00\:00%2B01\:00 TO *]
+```
+
+### creator
+
+Use the `creator` parameter to search for documents created by a specific creator.
+
+<!-- theme: warning --> 
+> At the time of writing, creator can contain a nickname or emailaddress. In the near future creator will only contain the uuid.
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers` `/organizers`
+
+**Possible values**
+
+Any uuid of a creator.
+
+**Example**
+
+Retrieve all events and places craeted by user with uuid `86a02c65-696d-5b12-a9b1-9e3bc8e6303c`:
+
+```
+GET /offers/?q=creator:86a02c65-696d-5b12-a9b1-9e3bc8e6303c
+```
+
+### dateRange
+
+Using the `dateRange` parameter, you can get all events and places that took place in a given range in the past, or will take place in a given range in the future.
+
+> By default, the search API will only return results that are currently available. In order to also retrieve results that are not available (yet), you'll need to disable the default filters for `availability`. You can reset this default as described in the Default Filters (TO DO: link to guide).
+
+**Applicable on endpoints** 
+
+`/events` `/places` `/offers` 
+
+**Possible values**
+
+Any ISO-8601 datetime, e.g. `2023-04-01T12:08:01+01:00`
+
+<!-- theme: info -->
+> Note that the `+` sign should be encoded for URLs (as `%2B`). Otherwise it will be interpreted as whitespace and the given date time will be considered invalid.
+
+**Example**
+
+Retrieve all events that took place in the year 2020:
+```
+GET /events/?q=dateRange:[2020-01-01T00\:00\:00%2B01\:00 TO 2021-12-31T23\:59\:59%2B01\:00]&availableTo=*&availableFrom=*
+```
+
+### \_exists_
+
+With the `_exists_` parameter you can search for the documents that have a specific property in their JSON body.
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers` `/organizers`
+
+**Possible values**
+
+Any property name from the UiTdatabank JSON scheme.
```

Comment:
(See other comment) Not all properties from the UiTdatabank JSON schemas, and some with different names. It can only be the field names listed in this document basically